test/config.test.ts

```diff
@@ -0,0 +1,172 @@
+import * as path from 'path';
+import fs from 'fs';
+import fstest from 'fs';
+import { getConfig, getFileConfig, getArgsConfig } from '../src/config';
+import { defaultConfig } from '../src/constants';
+import _, { includes } from 'lodash';
+
+describe('Config', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    process.argv = [];
+  });
+  describe('getConfig', () => {
```

Comment:
Let's add some spacing in this file? I suggest putting a line break before `describe` blocks and between `it` blocks, for example:
```suggestion
  });

  describe('getConfig', () => {
```